### PR TITLE
Sprint 7 merge 2

### PR DESCRIPTION
**Pull request by DanielvanVliet**
This PR contains the following issues:

- [Check storybook for incorrect MUI imports #33](https://github.com/Gemeente-DenHaag/denhaag-component-library/issues/33)
- [Storybook documentation is broken #23](https://github.com/Gemeente-DenHaag/denhaag-component-library/issues/23)
- [Fix relative imports #100](https://github.com/Gemeente-DenHaag/denhaag-component-library/issues/100)

### CODE REVIEW

File: src/stories/input/Switch.stories.tsx [hunk 1]
```diff
@@ -1,7 +1,7 @@
 import React from "react";
 import { Meta, Story } from "@storybook/react";
 import pkg from "../../components/Switch/package.json";
-import Switch, { SwitchProps } from "../../components/Switch/Switch";
+import Switch, { SwitchProps } from "../../components/Switch";
 import { FormControlLabel } from "../../components/FormControlLabel";
```
Comment (mjcarsjens) [inline src/stories/input/Switch.stories.tsx:5]: Why are we not using the default import here?

File: src/stories/input/DatePicker.stories.tsx [hunk 2]
```diff
@@ -1,8 +1,8 @@
 import React from "react";
 import { Story, Meta } from "@storybook/react";
+import DateFnsUtils from "@date-io/date-fns";
 import DatePicker, { DatePickerProps } from "../../components/Pickers/DatePicker/DatePicker";
 import { PickersUtilsProvider } from "../../components/PickersUtilsProvider/PickersUtilsProvider";
```
Comment (mjcarsjens) [inline src/stories/input/DatePicker.stories.tsx:5]: Shouldn't this one be changed as well?

(`../../components/PickersUtilsProvider`, and default import)

File: src/stories/input/ButtonGroup.stories.tsx [hunk 3]
```diff
@@ -2,7 +2,7 @@ import React from "react";
 import { Meta, Story } from "@storybook/react";
 import { ButtonGroup, ButtonGroupProps } from "../../components/ButtonGroup";
```
Comment (mjcarsjens) [inline src/stories/input/ButtonGroup.stories.tsx:3]: `import ButtonGroup, { ButtonGroupProps }`

File: src/stories/datadisplay/AvatarGroup.stories.tsx [hunk 4]
```diff
@@ -1,6 +1,6 @@
 import React from "react";
 import { Meta, Story } from "@storybook/react";
-import Avatar from "../../components/Avatar/Avatar";
+import Avatar from "../../components/Avatar";
 import { AvatarGroup, AvatarGroupProps } from "../../components/AvatarGroup";
```
Comment (mjcarsjens) [inline src/stories/datadisplay/AvatarGroup.stories.tsx:4]: `import AvatarGroup, { AvatarGroupProps }`

(at least that's how it's done for other components)

File: src/stories/layout/GridListTile.stories.tsx [hunk 5]
```diff
@@ -1,7 +1,7 @@
 import React from "react";
 import { Meta, Story } from "@storybook/react";
 import { GridList } from "../../components/GridList";
```
Comment (mjcarsjens) [inline src/stories/layout/GridListTile.stories.tsx:3]: default import

Comment (mjcarsjens) [inline src/stories/layout/GridListTile.stories.tsx:3]: default import

**Comment by bddjong** [comment]
@mjcarsjens your comments are addressed in this PR https://github.com/Gemeente-DenHaag/denhaag-component-library/pull/119

**Review by GewoonMaarten — APPROVED** [review]

**Review by bddjong — APPROVED** [review]

**Review by joostvanviegen — APPROVED** [review]

**Pull request merged**